src/components/purchase-complete/index.module.scss

```diff
@@ -31,6 +31,8 @@
   }
   button {
       margin: 10px 5px 0 5px;
+      background-color: var(--color_input_button_background_accent) !important;
+      color: var(--color_input_button_background_accent_text) !important;
```

Comment:
same here
like --**color_input_button_background_accent_text** to **color_button_color**?